Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG commit=develop
+FROM elifesciences/peerscout_client:${commit} AS client
```

Comment:
This is just a thought: whether rather than having a direct dependency to the client here, it would make sense to just mount the directory from the outside. The API could in theory be run without a client. In some distant future there will probably be other API consumers (xpub) that are not using the client (at this stage it's just speculation of course).

Reply:
It is possible at the docker level (it's just another volume) but it makes it difficult to ship a versioned artifact containing the client code. Long-term we could also push the `peerscout_client` image, rearchitecting it to make it also able to serve the client (with `nginx` or similar) rather than just containing the files.

Reply:
Okay, versioning is a good point. Better for them to be shipped together in the absence of API versioning.